core/arch/arm/plat-vexpress/platform_config.h

```diff
@@ -126,6 +126,11 @@
 
 #elif defined(PLATFORM_FLAVOR_qemu_armv8a)
 
+#ifdef CFG_CORE_SEL2_SPMC
+#define DRAM0_BASE             0x40000000
+#define DRAM0_SIZE             0xC0000000
```

Comment:
This is a runtime parameter for QEMU.

Reply:
Thanks for reviewing the patch. 
If you mean configure nsec_memory through external DT to the op-tee , can you share some example configuration?  
        memory@40000000 {
                device_type = "memory";
                reg = <0x40000000 0xC0000000>; /* non-secured memory */
        };  ?

Reply:
All the other QEMU configurations pick up the non-secure memory from the DTB provided at 0x40000000 by the emulator.